lib/routes.json

```diff
@@ -5347,7 +5360,13 @@
                 "$owner": null,
                 "$repo": null,
                 "$sha": null,
-                "$body": null,
+                "body": {
```

Comment:
This can be $body since it matches the global definition of type: String, required: true.